espnet2/enh/extractor/td_speakerbeam_extractor.py

```diff
@@ -118,10 +140,18 @@ def forward(
         B, L, N = feature.shape
 
         feature = feature.transpose(1, 2)  # B, N, L
-        aux_feature = aux_feature.transpose(1, 2)  # B, N, L'
+        if self.use_spk_emb:
```

Comment:
It's a bit difficult to follow/understand in what cases `use_spk_emb=True` if I didn't know speakerbeam. Can you add some introduction comment here or above ?

Reply:
I will add a comment above:
```python
# NOTE(wangyou): When `self.use_spk_emb` is True, `aux_feature` is assumed to be
# a speaker embedding; otherwise, it is assumed to be an enrollment audio.
if self.use_spk_emb:
...
```